pal/src/thread/thread.cpp

```diff
@@ -2820,3 +2820,32 @@ void GetCurrentThreadStackLimits(ULONG_PTR* lowLimit, ULONG_PTR* highLimit)
     *highLimit = (ULONG_PTR) stackbase;
 #endif
 }
+
+static thread_local ULONG_PTR s_cachedThreadHighLimit = 0;
+
+bool IsAddressOnStack(ULONG_PTR address)
```

Comment:
Discard this one. In that case (if system doesn't support limits etc.) `pthread_attr_getstack` should fail.